src/app/profile/email-verification/email-verification.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  ActivatedRoute,
+  Router
+} from '@angular/router';
+
+@Component({
+  selector: 'fabric8-cleanup',
+  templateUrl: 'email-verification.component.html',
+  styleUrls: ['./email-verification.component.less']
+})
+export class EmailVerificationComponent implements OnInit {
+
+  private verified: string = 'false';
+  private message: string = 'Your e-mail has been confirmed.';
+  private secMessage: string = 'Thank you for validating your e-mail address.' +
+  'You can now continue to use Openshift.io';
+
+  constructor(
+    private route: ActivatedRoute) {}
+
+  ngOnInit(): void {
+    const queryParams = this.route.snapshot.queryParams;
+    if (this.route.snapshot.queryParams['verified']) {
+      this.verified = this.route.snapshot.queryParams['verified'];
+      if (this.verified === 'false') {
```

Comment:
You may want to set verified to a boolean value? That way, you don't have to test strings in the component and HTML template. For example:

this.verified = (this.route.snapshot.queryParams['verified'] === 'true');